generators/app/templates/src/app/screens/Login/styles.scss

```diff
@@ -0,0 +1,22 @@
+.login-container {
+  height: 100vh;
+  width: 100%;
+}
+
+.button {
+  border: solid 1px;
+  border-radius: 5px;
+  height: 40px;
+}
+
+.input {
+  height: 30px;
+  border: solid 1px;
+  border-radius: 5px;
+}
+
+.form-container {
+  width: 100%;
```

Comment:
I don't think we need this. Maybe use item-1 as a class may fix this without this property